apps/SageAccountWeb/tsconfig.json

```diff
@@ -9,7 +9,32 @@
     "isolatedModules": true,
     "noFallthroughCasesInSwitch": true,
     "useUnknownInCatchVariables": false,
-    "types": ["vite/client", "vitest/globals", "@testing-library/jest-dom"]
+    "types": ["vite/client", "vitest/globals", "@testing-library/jest-dom"],
+    "paths": {
+      "@sage-bionetworks/synapse-client": [
+        "../../../packages/synapse-client/src"
+      ],
+      "@sage-bionetworks/synapse-client/*": [
+        "../../../packages/synapse-client/src/*"
+      ],
+      "@sage-bionetworks/synapse-types": [
+        "../../../packages/synapse-types/src"
+      ],
+      "@sage-bionetworks/synapse-types/*": [
+        "../../../packages/synapse-types/src/*"
+      ],
+      "synapse-react-client": ["../../../packages/synapse-react-client/src"],
+      "synapse-react-client/*": [
+        "../../../packages/synapse-react-client/src/*"
+      ],
+      "vite-config": ["../../../packages/vite-config/src"],
+      "vite-config/*": ["../../../packages/vite-config/src/*"]
+    }
```

Comment:
In each Vite project, adding these tsconfig `paths` properties informs the TypeScript compiler to use the sources of the projects rather than the built files. We combine this with a popular Vite plugin to achieve HMR across modules. You can start the Vite dev server for the app, make a change in synapse-react-client, and instantly see the change in the app without rebuilding.